carbon-reductor-core/src/main/java/de/envite/greenbpm/carbonreductor/core/domain/service/DelayCalculatorService.java

```diff
@@ -67,14 +67,13 @@ public CarbonReduction calculateDelay(CarbonReductorConfiguration input) throws
     }
 
     Timeshift calculateTimeshiftWindowForSLA(CarbonReductorConfiguration input) {
-        // maximumProcessDuration - milestone - remainingDuration - now
+        // lastPossibletEndDateTime - remainingDuration - now
         // (milestone + maximumDuration) - milestone - remaining - (now - milestone)
-        OffsetDateTime maximumDurationDateTime = input.getMilestone().asDate().plus(input.getMaximumProcessTimeshift().getValue().toMillis(), ChronoUnit.MILLIS);
-        long msSinceMilestone = OffsetDateTime.now(ZoneOffset.UTC).toInstant().toEpochMilli() - input.getMilestone().asDate().toInstant().toEpochMilli();
+		OffsetDateTime lastPossibleEndDateTime = input.getMilestone().asDate().plus(input.getMaximumProcessTimeshift().getValue().toMillis(), ChronoUnit.MILLIS);
```

Comment:
Formatting still has some potential here

Reply:
That is?

Reply:
I don't know why github shows an indention. My IDE doesn't